### PR TITLE
Added export-to-pdf-enable

### PR DESCRIPTION
**Pull request by theoldfather**
**Issue #, if available:**

fixes support request 6883627671

**Description of changes:**

This script enables Jupyter to export a notebook directly to PDF. nbconvert depends on XeLaTeX and several LaTeX packages that are non-trivial to install because tlmgr is not included with the texlive packages provided by yum.

**Testing Done**

- [x] Notebook Instance created successfully with the Lifecycle Configuration
- [x] Notebook Instance stopped and started successfully
- [x] Documentation in the script around any network access requirements
- [x] Documentation in the script around any IAM permission requirements
- [x] CLI commands used to validate functionality on the instance
- [x] New script link and description added to README.md

```
# Provide your commands here
git clone https://github.com/theoldfather/amazon-sagemaker-notebook-instance-lifecycle-config-samples.git
cd amazon-sagemaker-notebook-instance-lifecycle-config-samples/
git pull origin export-to-pdf
chmod -R +x scripts/export-to-pdf-enable/*.sh
./scripts/export-to-pdf-enable/on-create.sh 
./scripts/export-to-pdf-enable/on-start.sh 
# then go to Jupyter/Jupyter Lab and Export Notebook to PDF
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Lifecycle Configurations provide a mechanism to customize Notebook Instances via
 * [connect-emr-cluster](scripts/connect-emr-cluster) - This script connects an EMR cluster to the Notebook Instance using SparkMagic.
 * [disable-uninstall-ssm-agent](scripts/disable-uninstall-ssm-agent) - This script disables and uninstalls the SSM Agent at startup.
 * [execute-notebook-on-startup](scripts/execute-notebook-on-startup) - This script executes a Notebook file on the instance during startup.
+* [export-to-pdf-fix](scripts/export-to-pdf-fix) - This script fixes a problem in SageMaker Notebook instances where Jupyter fails to export a notebook directly to PDF.
```
Comment (jaipreet-s) [inline README.md:15]: Do you think `export-to-pdx-enable` is a better name? :-)

File: scripts/export-to-pdf-fix/on-create.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+# OVERVIEW
+# This script fixes a problem in SageMaker Notebook instances where Jupyter fails to export a notebook
+# directly to PDF. nbconvert depends on XeLaTeX and several LaTeX packages that are non-trivial to
+# install because `tlmgr` is not included with the texlive packages provided by yum.
+
```
Comment (jaipreet-s) [inline scripts/export-to-pdf-fix/on-create.sh:9]: We should explicitly document that this required internet access to fetch the binaries from the `ctan`

File: README.md (hunk 1 above)
Comment (theoldfather) [reply README.md]: Good idea. I've resolved it.

File: scripts/export-to-pdf-fix/on-create.sh (hunk 2 above)
Comment (theoldfather) [reply scripts/export-to-pdf-fix/on-create.sh]: Done.

**Pull request merged**